src/components/street-geo.js

```diff
@@ -113,6 +124,10 @@ AFRAME.registerComponent('street-geo', {
     const create3DtilesElement = () => {
       const google3dElement = document.createElement('a-entity');
       google3dElement.setAttribute('data-no-pause', '');
+      google3dElement.id = 'google3d';
+      if (data.enableClipping) {
+        google3dElement.setAttribute('obb-clipping', '');
```

Comment:
just to confirm, setting this attribute is what enables the `obb-clipping` component to "run" right?

Reply:
yes this adds the obb-clipping component to the 3dtiles which starts it running